dataland-frontend/src/components/resources/dataTable/cells/DataPointDisplayComponent.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <span>{{ content.displayValue.label }}</span>
+  <DocumentLink
+    label=""
+    :suffix="formattedPageNumber"
+    :download-name="content.displayValue.fileName"
+    :file-reference="content.displayValue.fileReference"
+    show-icon
+  />
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+import {
+  type MLDTDisplayComponentName,
+  type MLDTDisplayObject,
+} from "@/components/resources/dataTable/MultiLayerDataTableCellDisplayer";
+import DocumentLink from "@/components/resources/frameworkDataSearch/DocumentLink.vue";
+
+export default defineComponent({
+  name: "DocumentLinkDisplayComponent",
+  components: { DocumentLink },
+  props: {
+    content: {
+      type: Object as () => MLDTDisplayObject<MLDTDisplayComponentName.DataPointDisplayComponent>,
```

Comment:
For these datapoints with report we require to state what the quality of the datapoint is and offer the possibility for an optional comment. However, these informations are neither visible in the QA page nor in the view page. Shouldn't this have been part of this ticket? If not is there a ticket for this extension already in the backlog?

Reply:
DALA-2156 should handle that.

Reply:
However we could easily add the quality to the displayed string. That's a one liner.

Reply:
No that would overload the String - let's rather wait for DALA-2156.